docs/USAGE.md

```diff
@@ -9,6 +9,7 @@
 
 | Short name 	| Long name                     	| required? 	| Description                                                                                                               	|
 |------------	|-------------------------------	|-----------	|---------------------------------------------------------------------------------------------------------------------------	|
+| `-a`       	| `--abort_after_schema_export`    	| Optional  	| Aborts the application process.                                                                   	|
```

Comment:
I'd remove the "s" to "Aborts" here to be consistent with the other descriptions.
```suggestion
| `-a`       	| `--abort_after_schema_export`    	| Optional  	| Abort the application process.                                                                   	|
```